### ISSUE TITLE
New version release keeps on showing Pop up after installing new update

### ISSUE THREAD

**Issue by ManjunathaKaliwal** [rwbutler/Updates#19]
**Describe the bug**
Recently we have integrated updates framework, so last version working fine but recently observed issue for all the users. We have released new version 3.0.32 with release notes, 
Issue - All user updated app to 3.0.32 but still it's showing pop as new version available. please let us know the issue?

**Expected behavior**
Appstore url (https://apps.apple.com/in/app/health-glow/id998878431)

**Screenshots**
<img width="502" alt="Screenshot 2022-06-16 at 11 30 05 AM" src="https://user-images.githubusercontent.com/87514421/174002158-820c7d19-44d3-4d20-bff2-8fafa7bbe7fc.png">

**Smartphone (please complete the following information):**
 -  iOS
 - Version 15.4
 -  iPhone 11

**Additional context**

**Comment by pedrofneves** [comment]
+1

**Comment by rwbutler** [comment]
Hi @pedrofneves would you be willing to share your implementation?

**Comment by rwbutler** [comment]
Or @ManjunathaKaliwal?

**Comment by rwbutler** [comment]
I'd be happy to take a look

**Comment by pedrofneves** [comment]
@rwbutler of course: 

```
Updates.appStoreId = env.infoForKey(.appStoreID)
Updates.notifying = .always
Updates.comparingVersions = [.build, .patch, .minor, .major]
Updates.updateType = .hard
Updates.checkForUpdates { result in
    UpdatesUI.promptToUpdate(result, presentingViewController: self)
}
```
The users complain when we deploy a patch modified version, e.g.: from 1.2.0 to 1.2.1

**Comment by rwbutler** [comment]
@pedrofneves Am I right in thinking that you only want users to be notified when there is an increase in the patch number? In that case, the configuration you want is:

```
Updates.comparingVersions = .patch
```

The way things are configured in the sample above - your users will be notified every time the build number changes.

**Comment by pedrofneves** [comment]
No. The problem is when the users install the new patched version Updates keep prompting the users to update the same version. Exactly as described by ManjunathaKaliwal.

**Comment by CavalcanteLeo** [comment]
+1
